front/src/modules/project/components/AddOrEditProjectModal.tsx

```diff
@@ -25,6 +24,8 @@ import PictureUploader from 'applications/operationalStudies/components/Project/
 import type { ApiError } from 'common/api/baseGeneratedApis';
 import { SerializedError } from '@reduxjs/toolkit';
 import { getUserSafeWord } from 'reducers/user/userSelectors';
+import { useOsrdActions } from 'common/osrdContext';
+import { operationalStudiesConfSliceActionsType } from 'reducers/osrdconf/operationalStudiesConf';
```

Comment:
Could you add `type` to all other type imports of the file please :)
```suggestion
import type { operationalStudiesConfSliceActionsType } from 'reducers/osrdconf/operationalStudiesConf';
```

Reply:
Done